cli/src/commands/mock.ts

```diff
@@ -23,6 +23,10 @@ export default class Mock extends Command {
 
   static flags = {
     help: flags.help({ char: "h" }),
+    proxyBaseUrl: flags.string({
+      description:
+        "If set, the server would act as a proxy and fetch data from the given remote server instead of mocking it"
```

Comment:
```suggestion
        "If set, the server will act as a proxy and fetch data from the given remote server for all non-draft endpoints instead of mocking them"
```